src/JS_Modules/Models/InvolvementService.js

```diff
@@ -3,9 +3,7 @@ import { involvmentApiEndPoint, contentType } from './Utils.js';
 const sendComment = async (movieID, name, comment) => {
   const response = await fetch(`${involvmentApiEndPoint}comments`, {
     method: 'POST',
-    headers: {
-      'Content-Type': 'application/json',
-    },
+    headers: contentType,
```

Comment:
are you sure this wont cause an error

Reply:
Not at all, I tested it before uploading. It is the same way in all service calls. A fancy way for trying to DRY the code